base/common/StringRef.h

```diff
@@ -49,6 +49,8 @@ struct StringRef
 
     explicit operator std::string() const { return toString(); }
     constexpr explicit operator std::string_view() const { return {data, size}; }
+
+    const char * find(int c) const { return strchr(data, c); }
```

Comment:
Let's remove this function from here.
Firstly, it is incorrect as there are non zero terminated strings.